md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessAPI.java

```diff
@@ -278,9 +313,9 @@ RestWrapper update(@ModelAttribute("process")
      * @return restWrapper It contains an instance of Process newly added.
      */
     @RequestMapping(value = {"/", ""}, method = RequestMethod.PUT)
-    @ResponseBody public
-    RestWrapper insert(@ModelAttribute("process")
-                       @Valid Process process, BindingResult bindingResult, Principal principal) {
+    @ResponseBody
+    public RestWrapper insert(@ModelAttribute("process")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "insert" is 14 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)